README.rst

```diff
@@ -10,7 +10,7 @@ make creating exploratory map views effortless. Here's a crash course:
 
     import gmplot
 
-    gmap = gmplot.GoogleMapPlotter(37.428, -122.145, 16)
+    gmap = gmplot.GoogleMapPlotter(37.428, -122.145, 16, apikey=YOUR_GOOGLE_API_KEY)
```

Comment:
I'd recommend defining the API key as `apikey = '' # (your API key goes here)` right above this line, then simply passing `apikey=apikey` into `gmplot.GoogleMapPlotter()`. That way users who copy-paste this whole code block won't get an error if they don't replace `YOUR_GOOGLE_API_KEY` (and a blank API key works, albeit with limited functionality).